### ISSUE TITLE
bounding box display mode for maya_usd stages

### ISSUE THREAD

**Issue by dietchmatch** [Autodesk/maya-usd#1358]
It would be nice to have a bounding box display option in object display for stages.

**Comment by sepux** [comment]
Yes plus one on this one. Especially with bigger assets is nice to have a bbox representation of your stage. Like for example the Moana Island

**Comment by santosg87** [comment]
@sepux @dietchmatch not exactly an option showing up in object display - but on Maya 2023.3 and above, we have Display Layer capability for most USD objects - including the proxuShape node. this should allow for adding a stage onto a display layer, and then controling bounding box through that. :)
